src/corva/testing.py

```diff
@@ -50,3 +58,13 @@ def run(
                     cache_url=SETTINGS.CACHE_URL,
                 ),
             )
+
+        if app is None:
+            return
+
+        return service.run_app(
+            has_secrets=True,
+            app_key='test_app_key',
+            api_sdk=FakeApiSdk(secrets={'test_app_key': secrets or {}}),
```

Comment:
It's possible to pass no `secrets` with `has_secrets=True` and wise versa. Seems, like one piece is redundant

Reply:
This is for app testing. If user passes secrets - he will see them in tests, otherwise empty dict is set.

I could have written, but this seems more complex
```python
  has_secrets = secrets is not None
  api_sdk_secrets = {'test_app_key': secrets} if has_secrets else None

  return service.run_app(
      has_secrets=has_secrets,
      app_key='test_app_key',
      api_sdk=FakeApiSdk(secrets=api_sdk_secrets),
      app=app,
  )
```